### PR TITLE
Webpack: Support .cjs extension

### PR DESCRIPTION
**Pull request by ddalpange**
Issue: N/A

## What I did

I can't resolve my package that was made as commonjs .
I added to `.cjs` extension in `webpack rule -> test`

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: app/angular/src/server/framework-preset-angular-cli.test.ts [hunk 1]
```diff
@@ -793,7 +793,7 @@ const newWebpackConfiguration = (
       rules: [{ keepBaseRule: true } as any],
     },
     resolve: {
-      extensions: ['.mjs', '.js', '.jsx', '.ts', '.tsx', '.json', '.cjs'],
+      extensions: ['.cjs', '.mjs', '.js', '.jsx', '.ts', '.tsx', '.json'],
```
Comment (ndelangen) [inline app/angular/src/server/framework-preset-angular-cli.test.ts:796]: I fear this order changing will likely break some user's setup!

File: lib/builder-webpack4/src/preview/iframe-webpack.config.ts [hunk 2]
```diff
@@ -233,7 +233,7 @@ export default async (options: Options & Record<string, any>): Promise<Configura
       ],
     },
     resolve: {
-      extensions: ['.mjs', '.js', '.jsx', '.ts', '.tsx', '.json', '.cjs'],
+      extensions: ['.cjs', '.mjs', '.js', '.jsx', '.ts', '.tsx', '.json'],
```
Comment (ndelangen) [inline lib/builder-webpack4/src/preview/iframe-webpack.config.ts:236]: same here

File: lib/builder-webpack5/src/preview/iframe-webpack.config.ts [hunk 3]
```diff
@@ -234,7 +234,7 @@ export default async (options: Options & Record<string, any>): Promise<Configura
       ],
     },
     resolve: {
-      extensions: ['.mjs', '.js', '.jsx', '.ts', '.tsx', '.json', '.cjs'],
+      extensions: ['.cjs', '.mjs', '.js', '.jsx', '.ts', '.tsx', '.json', '.cjs'],
```
Comment (ndelangen) [inline lib/builder-webpack5/src/preview/iframe-webpack.config.ts:237]: same here

File: lib/manager-webpack4/src/presets/manager-preset.ts [hunk 4]
```diff
@@ -156,7 +156,7 @@ export async function managerWebpack(
       ],
     },
     resolve: {
-      extensions: ['.mjs', '.js', '.jsx', '.json', '.cjs', '.ts', '.tsx'],
+      extensions: ['.cjs', '.mjs', '.js', '.jsx', '.json', '.cjs', '.ts', '.tsx'],
```
Comment (ndelangen) [inline lib/manager-webpack4/src/presets/manager-preset.ts:159]: same here

File: lib/manager-webpack5/src/presets/manager-preset.ts [hunk 5]
```diff
@@ -160,7 +160,7 @@ export async function managerWebpack(
       ],
     },
     resolve: {
-      extensions: ['.mjs', '.js', '.jsx', '.json', '.cjs', '.ts', '.tsx'],
+      extensions: ['.cjs', '.mjs', '.js', '.jsx', '.json', '.cjs', '.ts', '.tsx'],
```
Comment (ndelangen) [inline lib/manager-webpack5/src/presets/manager-preset.ts:163]: same here

**Review by ndelangen — CHANGES_REQUESTED** [review]
I fear changing the order of these resolve configs will be a problem for other users...

**Review by shilman — APPROVED** [review]
LGTM. Thanks so much for doing this @ddalpange !!!

**Pull request merged**